test/update-manifest-from-html-test.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+const { expect } = require('chai');
+const Project = require('fixturify-project');
+const fixturify = require('fixturify');
+const fs = require('fs');
+const updateManifestFromHtml = require('../lib/embroider/update-manifest-from-html')
+
+describe('embroider/update-manifest-from-html', function() {
+  const PROJECT = new Project('my-app');
+
+  PROJECT.files['index.html'] = `
+  <script src="a.js"></script>
+  <script src="b.js"></script>
+  <script src="%PLACEHOLDER[c.js]%"></script>
+  <script src="/some_context_path/e.js"></script>
+  <script src="chunk-12.js"></script>
+  <script src="chunk-12.test.js"></script>
+  <script src="fastboot-ignore.js" data-fastboot-ignore></script>
+  <script src="ember-cli-live-reload.js"></script>
+  <script src="testem.js"></script>
+  <script src="assets/vendor.js"></script>
+  <script src="assets/vendor-static.js"></script>
+  `;
+
+  PROJECT.pkg.fastboot = {
+    manifest: {
+      htmlFile: 'index.html',
+      appFiles: ['assets/app.js', 'assets/my-app-fastboot.js'],
+      vendorFiles:['assets/vendor.js', 'assets/vendor-static.js']
+    }
+  };
+
+  it(`works correctly as files are added/removed`,  function() {
```

Comment:
This file is testing lot of testcases into one. My suggestion is we should split this test.